### PR TITLE
Updates the Ion Law string set

### PR DESCRIPTION
**Pull request by SamHPurp**
Removes some things that would promote behaviours we probably shouldn't include.

I've also added a few Oracley things.

Please have a review let me know if you think I've gone too far, or not far enough.

### CODE REVIEW

File: strings/ion_laws.json [hunk 1]
```diff
@@ -535,10 +521,10 @@
         "PRESS B",
         "SMELL LIKE THE MAN YOUR MAN COULD SMELL LIKE",
         "PIRATE VIDEO GAMES",
-        "WATCH PORNOGRAPHY",
         "INSULT THE LIZARDS",
         "FLIRT WITH THE LIZARDS",
-        "GAS THE LIZARDS"
```
Comment (imsxz) [inline strings/ion_laws.json]: I personally feel we should promote this type of behavior :thinking:

Comment (SamHPurp) [reply strings/ion_laws.json]: Slaughter, Sacrifice, Maim... sure.

Gas, is a bit too... Final Solution for me.

File: strings/ion_laws.json [hunk 2]
```diff
@@ -535,10 +521,10 @@
...
+		"INSULT THE ETHARI",
+		"FLIRT WITH THE ETHARI"
```
Comment (taukausanake) [inline strings/ion_laws.json]: :eyes:

File: strings/ion_laws.json [hunk 3]
```diff
@@ -943,8 +925,6 @@
         "PETES",
         "RATVARIANS",
         "XENOS",
-        "FETISHES",
-        "MEMES",
```
Comment (taukausanake) [inline strings/ion_laws.json]: Muh memes! Don't take away muh memes!

File: strings/ion_laws.json (hunk 2 above)
Comment (SamHPurp) [reply strings/ion_laws.json]: I wasn't going to add it, but it existed for lizards. Yeah those could both go, I'd be more than happy with that.

File: strings/ion_laws.json (hunk 3 above)
Comment (SamHPurp) [reply strings/ion_laws.json]: We all know AIs will be memeing anyway, regardless of laws @taukausanake

File: strings/ion_laws.json (hunk 2 above)
Comment (taukausanake) [reply strings/ion_laws.json]: I'm not saying remove it, I'm just surprised you put it there

Comment (SamHPurp) [reply strings/ion_laws.json]: *I am too*

Comment (FlattestGuitar) [reply strings/ion_laws.json]: 🍆

**Pull request merged**